src/blocks/CTA/CTA.scss

```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
 
     &__button {
         display: flex;
-        padding-bottom: $indentXS;
+        padding-bottom: 0px;
+
+        &_layout {
+            width: 100%;
+            max-width: 100%;
+            margin-right: 0px;
+
+            &_2 {
```

Comment:
Can we set styles with nth-child?
isn't that strange that we named classes like numbers?

Reply:
I got rid of such classes at all. Thank you for the comment